src/main/resources/application-dev.yml

```diff
@@ -45,4 +43,4 @@ app:
     region: eu-west-2
     bucket-name: cjsm-orgs
   notify:
-    apikey: dummy-api-key
```

Comment:
is this change intended?

Reply:
thanks good spot. Needs to be dummy-api-key